crates/bevy_render/Cargo.toml

```diff
@@ -44,6 +44,7 @@ bevy_render_macros = { path = "macros", version = "0.11.0-dev" }
 bevy_time = { path = "../bevy_time", version = "0.11.0-dev" }
 bevy_transform = { path = "../bevy_transform", version = "0.11.0-dev" }
 bevy_window = { path = "../bevy_window", version = "0.11.0-dev" }
+bevy_winit = { path = "../bevy_winit", version = "0.11.0-dev" }
```

Comment:
This should be made into an optional (but on by default) dependency, behind a `winit` feature flag.